src/components/scenes/SendScene.js

```diff
@@ -429,6 +447,33 @@ class SendComponent extends React.PureComponent<Props, State> {
     return infoTiles.map(({ label, value }) => <Tile key={label} type="static" title={label} body={value} />)
   }
 
+  renderAuthentication() {
+    const { authRequired, pin, theme } = this.props
+    const styles = getStyles(theme)
+
+    if (authRequired === 'pin') {
+      return (
+        <Tile type="touchable" title={s.strings.four_digit_pin} onPress={this.handleFocusPin}>
+          <PinDots pin={pin} />
```

Comment:
Pass in the constant from the previous commit as a prop for the PinDots component.